src/vmspawn/vmspawn.c

```diff
@@ -697,6 +700,53 @@ static int setup_notify_parent(sd_event *event, int fd, int *exit_status, sd_eve
         return 0;
 }
 
+static int bus_open_in_machine(sd_bus **ret, unsigned cid, unsigned port, const char *private_key_path) {
```

Comment:
This is `static` and unused, so the compilation will at least cause a warning. Please merge this the commit that adds the use.